src/buf/fixed/registry.rs

```diff
@@ -73,7 +125,7 @@ impl FixedBufRegistry {
     /// of the `tokio-uring` runtime this call is made in, the function returns
     /// an error.
     pub fn register(&self) -> io::Result<()> {
-        crate::io::register_buffers(&self.inner)
+        crate::io::register_buffers(Rc::clone(&self.inner) as _)
```

Comment:
`<Rc as Clone>::clone` is shallow, it only increases the reference count. The method is invoked in type-qualified way to highlight this. The `as _` part is needed to prevent type inference from choosing the wrong clonee type, as references to different types of `Rc` cannot be coerced to one another, but the `Rc` instances themselves do coerce to unsized. This quirk is explained [here](https://stackoverflow.com/a/63894064/2217151).